src/Cli/Microsoft.TemplateEngine.Cli/Commands/details/BaseDetailsCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Cli.NuGet;
+using Microsoft.TemplateEngine.Edge.Settings;
+
+namespace Microsoft.TemplateEngine.Cli.Commands
+{
+    internal class BaseDetailsCommand : BaseCommand<DetailsCommandArgs>
+    {
+        private static NugetApiManager _nugetApiManager = new NugetApiManager();
+
+        internal BaseDetailsCommand(
+            Func<ParseResult, ITemplateEngineHost> hostBuilder,
+            string commandName)
+            : base(hostBuilder, commandName, SymbolStrings.Command_Details_Description)
+        {
+            AddArgument(NameArgument);
+            AddOption(VersionOption);
```

Comment:
should we support
- getting details from the feed other than NuGet.org
- interactive auth

@baronfel what do you think?

This would make the command consistent with other usages.

Reply:
yes - details, unlike search, should be usable for any package from any feed. It should also accept the interactive flag to allow for authentication.

Apologies for the delay here!